benchmarks/benchmarks_stats.py

```diff
@@ -0,0 +1,34 @@
+"""Benchmarks suite for stats module."""
+
+import numpy as np
+import xarray as xr
+
+from sgkit import count_call_alleles, count_cohort_alleles
+from sgkit.tests.test_aggregation import get_dataset
```

Comment:
I think you could use `simulate_genotype_call_dataset` directly (which is public), rather than `get_dataset`.

Reply:
Ah, I see thanks! didn't realise that.